lib/pg_online_schema_change/orchestrate.rb

```diff
@@ -159,6 +164,12 @@ def setup_trigger!
       end
 
       def setup_shadow_table!
+        logger.info("Setting up shadow table", { shadow_table: shadow_table })
+        Query.run(
+          client.connection,
+          "SELECT create_table_all('#{client.table_name}', '#{shadow_table}');",
+        )
+
```

Comment:
Noting: This bit. I believe its ok to create the shadow table outside of serializable transaction. All the following stuff that is running the alter statement, deleting rows from audit table and then copying the data from audit table into shadow table - is all happening in the single transaction. Thats the important bit to ensure no duplicates are copied over. 

And by doing this, it allows us to show estimated progress as well.